scripts/simplify_network.py

```diff
@@ -838,7 +838,7 @@ def merge_isolated_nodes(n, threshold, aggregation_strategies=dict()):
     n = pypsa.Network(snakemake.input.network)
 
     base_voltage = snakemake.params.electricity["base_voltage"]
-    linetype = snakemake.params.config_lines["types"][base_voltage]
+    linetype = snakemake.params.config_lines["ac_types"][base_voltage]
```

Comment:
What would happen if `base_voltage` in `electricity` section of the config doesn't correspond to any of the standard voltage values defined in `ac_types`? Could it be possible to add some back-up approach?

Reply:
I think that may not be needed; if we receive complaints, we can revise

Reply:
Agree, it may be better not to over-complicate :)